content/news/2017-HLSPs.md

```diff
@@ -0,0 +1,25 @@
+Title: New and Updated K2 High Level Science Products
+Date: 2017-08-16 12:00
+Author: Ann Marie Cody
+
+The K2 communinity has been very active in working with the mission's images and light curves, and a number of groups have made 
+their output available to the public in the form of High Level Science Products. We are happy to report updates to several of these 
+submissions over the last 6 months. The K2 Mission encourages further submissions of HLSPs and our Guest Observer Office is happy 
+to help astronomers interface with the MAST archive.
+
+This month, the [K2 Systematics Correction](https://archive.stsci.edu/prepds/k2sc/) ("K2SC") by Aigrain et al. is celebrating the 
+release of version 2, with light curves now available through Campaign XX. These data products provide time series detrended for 
+positional jitter and/or time-dependent systematics. As such, they are well suited to studying variable stars. 
```

Comment:
"they are well suited to" => "they are intended to be well suited to" (NASA English)